src/components/ThreadEnvelope.vue

```diff
@@ -60,6 +61,9 @@
 				@click.native.prevent="$emit('toggle-expand', $event)">
 				<div class="sender">
 					{{ envelope.from && envelope.from[0] ? envelope.from[0].label : '' }}
+					<p class="sender-email">
```

Comment:
```suggestion
					<p class="envelope__header__sender-email">
```

for the BEM convention